movie_REST_API/movie_REST_API/App/Model/MoviesData.swift

```diff
@@ -0,0 +1,53 @@
+// MoviesData.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import SwiftyJSON
+
+/// Moдель массив фильмов
+struct MoviesData: Decodable {
+    /// массив с фильмами
+    var movies: [Movie]
+
+    private enum CodingKeys: String, CodingKey {
+        case movies = "results"
+    }
+}
+
+/// Moдель фильмы
+struct Movie: Decodable {
+    /// идентификатор фильма
```

Comment:
```suggestion
    /// идентификатор фильма
```
С большой буквы дока на все поля